src/theme/components/toggle.ts

```diff
@@ -0,0 +1,66 @@
+import { helperTextMargin } from "./global";
+
+// Style object for the Checkbox's label
+const baseStyleLabel = {
+  fontSize: "sm",
```

Comment:
This works but should be `"-1"` (I think). The values to use for font sizes are in the `typography.ts` file.